wled00/wled20_ir.ino

```diff
@@ -68,8 +76,16 @@ void decodeIR(uint32_t code)
 
   if (decodeIRCustom(code)) return;
   if      (code > 0xFFFFFF) return; //invalid code
-  else if (code > 0xFF0000) decodeIR44(code); //is in 44-key remote range
   else if (code > 0xF70000 && code < 0xF80000) decodeIR24(code); //is in 24-key remote range
+  else if (code > 0xFF0000) {
+    switch (irEnabled) {
+      case 1: decodeIR24OLD(code); break;  // white 24-key remote (old) - it sends 0xFF0000 values
+      case 2: decodeIR24CT(code);  break;  // white 24-key remote with CW, WW, CT+ and CT- keys
+      case 3: decodeIR40(code);    break;  // blue  40-key remote with 25%, 50%, 75% and 100% keys
+      case 4: decodeIR44(code);    break;  // white 44-key remote with color-up/down keys and DIY1 to 6 keys 
+      default: return;
+    }
+  }
```

Comment:
the "new" IR remotes start with "0xFF0000" (with NEC protocol).
To switch between them configure the IR remote value in the settings.